theme/theme.css

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
What's this file? I think it's not used at all.

Reply:
Or it is... https://github.com/ckeditor/ckeditor5-build-balloon-block/pull/2/files#diff-61c6b4c84ae09dd9e9b7a0577466fd28R32

But why is it defined in a build and not somewhere by default?

Reply:
Because by default the block toolbar positions next to the editable. It's up to integration to configure it (editable can have border/background or not, we don't know it). I figured by default we should provide such button placement it works out of the box for all integrations. And hence this file.